projects/mac_ai/visualizations/llm_load_test/store/lts_parser.py

```
@@ -9,6 +9,22 @@
 from ..models import lts as models_lts
 
 
+def _generate_inter_token_latency(results):
+    if not results.llm_load_test_output: return None
```

Ruff (0.14.0)

Multiple statements on one line (colon)

(E701)